locales/az_AZ/LC_MESSAGES/duckduckgo.po

```diff
@@ -4077,9 +4071,8 @@ msgctxt "reasons-to-use-duckduckgo"
 msgid "We don't track you. Ever."
 msgstr "Biz sizi izləmə. Hələ."
 
-#, fuzzy
 msgid "We don\\'t track you."
-msgstr "Biz sizi izləmirik."
```

Comment:
Comparing with the above translations for this string, it seems translations for `We don\\'t track you` might be correct even though it's marked as fuzzy?

I think we've got a token like `We don't track you` (no `\\`), which is why it's correct. I wonder if we need both.